collect_app/src/test/java/org/odk/collect/android/formmanagement/FormMetadataParserTest.java

```diff
@@ -37,7 +36,7 @@ public void canParseFormWithExternalSecondaryInstance() throws Exception {
         File externalInstance = new File(mediaDir, "external-data.xml");
         FileUtils.write(externalInstance, EXTERNAL_INSTANCE.getBytes());
 
-        FormMetadataParser formMetadataParser = new FormMetadataParser(referenceManager);
```

Comment:
I think we also want to remove the `referenceManager` field from the test class as well right? There are tests that use it that happen to pass right now but they're definitely misleading.